tests/integrations/realcluster/reboot_pd_test.go

```diff
@@ -14,61 +14,55 @@
 
 package realcluster
 
-import (
-	"context"
-	"os/exec"
-	"testing"
+// func restartTiUP() {
```

Comment:
How about using the build flag `enable_flaky_tests`?

Reply:
I'm not quite sure how `enable_flaky_tests` works. I reverted the changes to this file first because the test is just not working, not because it's unstable. https://github.com/tikv/pd/pull/8303/commits/ad19424c860167302b3b0146900eecc57391d0d7

Reply:
It's not a flaky test, can be fixed by https://github.com/tikv/pd/pull/8303#discussion_r1647216610

Reply:
I fixed it. https://github.com/tikv/pd/pull/8303/commits/fa47700c0aa37c4d5302c9a91217aed4e489b7f4

Reply:
got